web/turkey/survey/static/survey/js/auditors/keypresses_specific.js

```diff
@@ -0,0 +1,24 @@
+var AuditorKeypressesSpecific = {
+    keypresses_specific: [],
+    log_keypress_content: function (e) {
+        this.keypresses_specific.push(String.fromCharCode(e.which));
```

Comment:
Forgot to change it before making PR.

Respectively, `\b`, `\r`, `\u000e or \u000f`, `\t`.